qa/rpc-tests/sc_cert_base.py

```diff
@@ -250,7 +250,7 @@ def run_test(self):
             errorString = e.error['message']
             mark_logs(errorString, self.nodes, DEBUG_MODE)
 
-        assert_equal("bad-sc-cert-not-applicable" in errorString, True)
+        assert_equal("bad-sc-cert-proof" in errorString, True)
```

Comment:
Since `bad-sc-cert-proof` is used several time, consider storing it in a (const) variable.